aggregator/elem.go

```diff
@@ -360,6 +335,15 @@ func (e *CounterElem) processValue(timeNanos int64, agg aggregation.Counter, fn
 	}
 }
 
+// NB(cw) suffix overrides default suffixes to not add
+// suffix for default Counter metrics for backward compatibility reasons.
+func (e *CounterElem) suffix(aggType policy.AggregationType) []byte {
+	if aggType == policy.Sum {
```

Comment:
I don't think it should be set in the aggregator options, because the assumption was also made in the aggregation package